adapter/internal/discovery/xds/server.go

```diff
@@ -602,10 +608,13 @@ func DeleteAPIWithAPIMEvent(uuid, organizationID string, environments []string)
 		} else {
 			// if no error, update internal vhost maps
 			// error only happens when API not found in deleteAPI func
-			logger.LoggerXds.Infof("Successfully undeployed API %v of Organization %v from environments %v", apiIdentifier, organizationID, environments)
+			logger.LoggerXds.Infof("Successfully undeployed API %v of Organization %v from environments %v revision %v before revisionUUID", apiIdentifier, organizationID, environments, revisionUUID)
```

Comment:
meaning of ` revision %v before revisionUUID` confused me. what exactly are we trying to say. is it `Revision xxxx of API Petstore:1.0.0 in org foo was undeployed successfully from abc, def environments`?

Reply:
Ah sorry, that was a mistakenly committed. `before revisionUUID` this should be removed.

Reply:
Fixed via : https://github.com/wso2/product-microgateway/pull/2503/commits/4d77d188a4aefb79e5d8582c06613d46ae0a1348